docs/GettingStartedOnLinux.md

```diff
@@ -0,0 +1,217 @@
+---
+id: getting-started
+title: Getting Started on Linux
+layout: docs
+category: Quick Start
+permalink: docs/getting-started-on-linux.html
+next: android-setup
+---
+
+## Intro
+
+This guide is essentially a beginner-friendly version of the [Getting Started](/react-native/docs/getting-started.html) page for React Native on Linux.
+
+### Platform
+
+For the purposes of this guide, we assume that you're working on Ubuntu Linux 14.04 LTS.
+
+### Prerequisites
+
+Before following this guide, you should have installed the Android SDK and run a successful Java-based "Hello World" app for Android.
```

Comment:
Let's merge the "Platform" and "Prerequisites" to one section. Also we can link to `/react-native/docs/android-setup.html` here.

```
This guide assumes that you're working with Ubuntu Linux 14.04 LTS. Other versions of Ubuntu might work, but not tested. Before continuing, be sure to [install the Android SDK](/react-native/docs/android-setup.html).
```